Annotato/Annotato/ViewModels/Document/DocumentViewModel.swift

```diff
@@ -15,10 +16,32 @@ class DocumentViewModel: ObservableObject {
         guard let baseFileUrl = URL(string: model.baseFileUrl) else {
             return nil
         }
-
         self.model = model
         self.pdfDocument = PdfViewModel(baseFileUrl: baseFileUrl)
         self.annotations = model.annotations.map { AnnotationViewModel(model: $0, document: self) }
+        setUpSubscribersForAllAnnotations()
+    }
+
+    private func setUpSubscribersForAllAnnotations() {
```

Comment:
```suggestion
    private func setUpSubscribers() {
```

When individual annotations are added, we set up subscribers for them separately

Reply:
Ok done too